deploy/examples/operator.yaml

```diff
@@ -646,6 +646,10 @@ spec:
             - name: ROOK_UNREACHABLE_NODE_TOLERATION_SECONDS
               value: "5"
 
+            #  Custom label to identify node hostname. If not set `kubernetes.io/hostname` will be used
+            - name: ROOK_CUSTOM_HOSTNAME_LABEL
+              value: ""
```

Comment:
@travisn is it makes sense to comment out envar instead?

Reply:
This is fine to leave it empty and uncommented. It is easier to notice this way.